### ISSUE TITLE
Disable Dependabot after renovate trial

### ISSUE THREAD

**Issue by cevich** [containers/netavark#587]
With https://github.com/containers/netavark/pull/526 merged, both renovate and dependabot are operating on this repo.  Assuming things go well, (they both find the same set of dep. updates) once this issue goes stale, dependabot may be disabled.  That includes BOTH the settings and any `.github/dependabot.yml` configuration file.

**Comment by cevich** [comment]
Note: Upon examining open dep-update PRs, I see a massive difference between renovate and dependabot.  For example, `chronotope/chrono` has a dependabot PR for `0.4.24` but recent renovate logs indicate `0.4.22` is the "latest":

```json
          {
            "depName": "chrono",
            "depType": "build-dependencies",
            "currentValue": "0.4.22",
            "managerData": {
              "nestedVersion": true
            },
            "datasource": "crate",
            "updates": [],
            "packageName": "chrono",
            "warnings": [],
            "versioning": "cargo",
            "sourceUrl": "https://github.com/chronotope/chrono",
            "registryUrl": "https://crates.io",
            "dependencyUrl": "https://crates.io/crates/chrono",
            "currentVersion": "0.4.22"
          },
```

investigating further...

**Comment by Luap99** [comment]
I have no idea.
The important thing to know about cargo is that most versions in Cargo.toml are not fixed, e.g. the version listed != version used. Cargo.lock contains the actual versions and this lists chrono 0.4.23.

Looking at the number of open PRs it seems that renovate is definitely missing more than that. Maybe they check a different source or at a different interval?

**Comment by cevich** [comment]
Oh okay, [the renovate docs for rust explain it better](https://docs.renovatebot.com/rust/#how-it-works).  That also maybe explains the discrepancy - maybe there's a delay in crates.io picking up the new versions whereas dependabot is using repo. tags or releases.

**Comment by cevich** [comment]
Ugh, nope.  https://crates.io/crates/chrono shows `0.4.24` as the latest, as of 9 days ago :disappointed: Damn.

**Comment by cevich** [comment]
Opened https://github.com/renovatebot/renovate/discussions/21194

**Comment by cevich** [comment]
Problem fixed with https://github.com/containers/automation/commit/7861f60698ad6dea86775b4007e4e9115b0ce731

**Comment by Luap99** [comment]
> Problem fixed with [containers/automation@7861f60](https://github.com/containers/automation/commit/7861f60698ad6dea86775b4007e4e9115b0ce731)

confirmed: https://github.com/containers/netavark/pull/650
